frontend/packages/shared/src/styles/abstracts/_mixins.scss

```diff
@@ -52,8 +52,8 @@
   }
 }
 
-@mixin desktop {
-  @media (max-width: map-get(maps.$breakpoints, 'desktop')) {
+@mixin desktop-md {
+  @media (min-width: map-get(maps.$breakpoints, 'desktop-xs')) and (max-width: map-get(maps.$breakpoints, 'desktop-lg')) {
```

Comment:
Olikos niin, että clerk puolta ei tarvi pystyä mobiililla käyttämään mutta public kyllä? Aika pienellä tyylityksellähän tässä on selvitty mutta oisko noita media queryitä hyvä teoriassa rakentaa pienemmästä ylöspäin? Jos olisi esim. `desktop-xs-and-up`, `desktop-md-and-up` ja `desktop-lg-and-up` ja niillä min-width? Nimet voi vielä miettiä mut tuolla saisi yhdellä rivillä targetoitua kaikkia desktop koot ja jos tarvetta niin sitten lisätä perään noita isompien kokojen stylejä. Nyt hieman jännä kun vain `desktop-md` välillä 920 - 1680px. Näitä tyylityksiä ei käytetä 1680px leveämmällä ruudulla?

Reply:
Joo tosiaan siellä olevia tyylityksiä on tarkoitus käyttää vain välillä 920 - 1680px.

Reply:
Clerk UI:n ei tarvi olla mobiiliystävällinen, mutta sen ei pitäisi näkyä huonosti, kun resoluutio ei ole hyvä. Tämä koskee lähinnä Desktop laitteita

Reply:
Miltäs kuulostaisi tässä tapauksessa `desktop-xs-to-md` tai `desktop-xs-md`?

Reply:
Mjoo, xs-to-md selkeämpi tuossa. Tosin jos se menee 1680 asti ni pitäskö sen olla xs-to-lg itseasiassa?

Reply:
joo totta, hyvä huomio. Vaihdetaan.

Reply:
Vaihdettu